Nos/Models/NoteParser.swift

```diff
@@ -110,12 +118,19 @@ struct NoteParser {
     }
     // swiftlint:enable function_body_length
 
-    /// Replaces Nostr entities embedded in the note (without a proper tag) with markdown links
-    private func replaceNostrEntities(in content: String) -> String {
+    /// Replaces Nostr entities embedded in the note (without a proper tag) with markdown links and
+    /// optionally extracts the first quoted note id.
+    ///
+    /// - Parameters:
+    ///   - content: The note content in which to replace entities.
+    ///   - capturesFirstNote: If true, this function will extract the first quoted note id, if it exists.
```

Comment:
We've been adding default values to doc comments like this:

```suggestion
    ///   - capturesFirstNote: If true, this function will extract the first quoted note id, if it exists. Defaults to `false`.
```